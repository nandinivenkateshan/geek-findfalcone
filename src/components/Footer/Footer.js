import React from 'react'
import styled from 'styled-components'

const StyledFooter = styled.footer`
text-align: center;
font-size:20px
`
const StyledLink = styled.a`
text-decoration:none;
color:#0d0f0e;
`

function Footer () {
  return (
    <StyledFooter>
        <p>
        Coding Problem - <StyledLink href="https://www.geektrust.in/" target='new'>www.geektrust.in/finding-falcone</StyledLink>
        </p>
    </StyledFooter>
  )
}

export default Footer
