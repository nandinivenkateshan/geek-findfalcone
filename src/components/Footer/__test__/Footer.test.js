import React from 'react'
import { render, cleanup } from '@testing-library/react'
import Footer from '../Footer'

afterEach(cleanup)

it('renders', () => {
  const { asFragment } = render(<Footer />)
  expect(asFragment()).toMatchSnapshot()
})
