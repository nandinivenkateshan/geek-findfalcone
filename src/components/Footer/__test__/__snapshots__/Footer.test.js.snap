// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <footer
    class="sc-AxjAm hvLVfH"
  >
    <p
      class="sc-AxiKw eXQrcs"
    >
      <span>
        Coding Problem -
      </span>
       
      <a
        class="sc-AxirZ jOWcRa"
        href="https://www.geektrust.in/"
        target="new"
      >
        www.geektrust.in/finding-falcone
      </a>
    </p>
  </footer>
</DocumentFragment>
`;
